src/applications/gi/reducers/profile.js

```diff
@@ -17,9 +17,8 @@ function normalizedAttributes(attributes) {
   const name = attributes.name
     ? attributes.name.toUpperCase()
     : attributes.name;
-  const city = attributes.city
-    ? attributes.city.toUpperCase()
-    : attributes.city;
+  let city = attributes.physicalCity || attributes.city;
```

Comment:
not clear to me that physical city is always available so using this as a fallback.